.github/maintainers_guide.md

```diff
@@ -0,0 +1,93 @@
+# Maintainers Guide
+
+This document describes tools, tasks and workflow that one needs to be familiar with in order to effectively maintain
+this project. If you use this package within your own software as is but don't plan on modifying it, this guide is
+**not** for you.
+
+## Tools
+
+All you need to work with this project is a supported version of [Node.js](https://nodejs.org/en/)
+(see `package.json` field "engines") and npm (which is distributed with Node.js).
+
+## Tasks
+
+### Testing
+
+This package has unit tests for most files in the same directory the code is in with the suffix `.spec` (i.e. `exampleFile.spec.ts`). You can run the entire test suite using the npm script `npm test`. This command is also executed by Travis, the continuous integration service, for every Pull Request and branch. The coverage is computed with the `codecode` package. The tests themselves are run using the `mocha` test runner.
+
+Test code should be written in syntax that runs on the oldest supported Node.js version, without transpiling. This ensures that backwards compatibility is tested and the APIs look reasonable in versions of Node.js that do not support the most modern syntax.
+
+A useful trick for debugging inside tests is to use the Chrome Debugging Protocol feature of Node.js to set breakpoints and interactively debug. In order to do this you must run mocha directly. This means that you should have already linted the source (`npm run lint`), manually. You then run the tests using the following command: `./node_modules/.bin/mocha test/{test-name}.js --debug-brk --inspect` (replace {test-name} with an actual test file).
+
+### Generating Documentation
+
+The documentation is built using [Jekyll](https://jekyllrb.com/) and hosted with GitHub Pages.
+The source files are contained in the `docs` directory. They are broken up into the `_basic`, `_advanced`, and `_tutorials` directories depending on content's nature.
+
+All documentation contains [front matter](https://jekyllrb.com/docs/front-matter/) that indicates the section's title, slug (for header), respective language, and if it's not a tutorial it contains the order it should appear within it's respective section (basic or advanced).
```

Comment:
Also, the maintainers need to be aware of also updating Japanese translation (plus we may have other languages as well in the future) when they update documents. 

I'm still not sure what the most easy-to-understand way to communicate regarding this matter for contributors is but one thing I can suggest is to have a more comprehensive checklist with categories (e.g., code change, documents, etc)  in `.github/pull_request_template.md`.

Reply:
Yeah, I think `pull_request_template.md` may be a good place to indicate this. Maybe we could add a label for documentation that is `Needs Translation`, but I also don't want to wait to merge PRs that aren't translated yet since that could take some time.